receiver/prometheusremotewritereceiver/README.md

```diff
@@ -0,0 +1,12 @@
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [beta]: metrics   |
+| Distributions | [core], [contrib] |
```

Comment:
We should leave this empty until it is added to the corresponding distributions.